theme/connector/view/_Learn.scss

```diff
@@ -941,3 +941,33 @@ img.detail-has-data-very-small {
     text-transform: uppercase;
   }
 }
+
+/* Data export button, with light arrow icon */
+.x-btn-rounded-inverted-accent-small .x-btn-arrow-light {
```

Comment:
Looks like the spec is expecting the arrow to point up when menu is expanded. Is that handled as well?

Reply:
It's not right now, I was going to ask the client if that were a must have versus nice to have since it doesn't easily fall out of the Ext4 implementation. I'm sure I could do it but would add more CSS complexity.

Reply:
implemented the up arrow when the menu is opened...